dataprofiler/data_readers/csv_data.py

```diff
@@ -231,7 +236,61 @@ def _guess_header_row(data_as_str, suggested_delimiter=None,
         # If change in statistics never occurs, return no header
         if not change_flag:
             row_classic_header_ends = None
+
+
+        # Attempt to resolve case where only strings in every cell
+        if only_string_flag:
```

Comment:
not sure I understand this block. Why the existing code doesn't work for the files with only strings?

Reply:
We discussed in chat, the logic currently used checks to see if there are changes in things such as: data type, data format, none counts, etc. Unfortunately, if you have a file of all strings none of the above can be applied. Instead, I utilized a method which checked the word counts per column and the difference down the column that is being evaluated. It appears effective at identifying the files I tested and also doesn't negatively impact the non-all-string files I tested.